client/src/main/java/cloud/prefab/client/config/ConfigResolver.java

```diff
@@ -255,6 +255,19 @@ List<EvaluatedCriterion> evaluateCriterionMatch(
     switch (criterion.getOperator()) {
       case ALWAYS_TRUE:
         return List.of(new EvaluatedCriterion(criterion, true));
+      case IN_TIME_RANGE:
+        if (prop.isEmpty() || !criterion.getValueToMatch().hasTimeRange()) {
+          return List.of(new EvaluatedCriterion(criterion, false));
+        }
+        long currentTime = prop.get().getInt();
```

Comment:
so there's nothing here that is actually _time_ specific.  As is this is really "in int range".  

That's just a statement, not a judgment.  

In a world where we share the operator with users: 

I guess the utility of an operator that says "time" is that it's a hint to the UI to show time widgets. If this is just IN_INT_RANGE, it's not obvious how to decide what UI to give a user. 

If we did then want to offer people an int range, we could do:

```java
case IN_INT_RANGE, IN_TIME_RANGE:
  ... code here that just talks about ints
```
Ah, but we'd want a different ConfigValue too. 

I guess my only thought I have an inclination not to call this `currentTime`. That seems presumptuous. Maybe it's a different time that isn't current. That's for the passed in property to know.